framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -320,6 +319,9 @@ def __init__(
         self.encode_types['uint64_t'] = 'UInt64'
 
         # Command parameter and struct member data for the current feature
+        self.struct_names = set()                              # Set of current API's struct typenames
+        self.extended_structs = dict()                         # Map of all extended struct names
```

Comment:
I end up switching it in the future.  So I'll leave the comment now.  This was everywhere and is backwards in my opinion because we typically used it so that we would say "what extends this struct".  So before, we looped through everything and saw if the modifided struct was in the list of any of the extended structs.